docs/input/docs/contributing/how-to-contribute.md

```diff
@@ -0,0 +1,19 @@
+---
+Order: 10
+Title: How to contribute
+Description: Instructions how to contribute to Cake.Recipe..
+---
+The repository is using [GitFlow] with default configuration.
+Development is happening on `develop` branch.
+
+To contribute:
+
+* Fork this repository.
+* Create a feature branch from `develop`.
+* Implement your changes.
+* Push your feature branch.
+* Create a pull request.
+
+For getting started see issues marked with `Up-for-grabs` in the individual repositories.
```

Comment:
We only have 1 repository for Cake.Recipe, and we don't really use the `Up for grabs` label.
I haven't seen it used at least.